CHANGELOG.md

```diff
@@ -6,6 +6,22 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## Unreleased
 
+### Upgrade Notes
+
+### Known Issues
+
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+## [0.6.0] - 2021-09-30
+
 Add the [Close Stale Issues](https://github.com/marketplace/actions/close-stale-issues) action
```

Comment:
can this be moved to the `Added` header?

Reply:
edit: nvm probably doesn't make sense to add explicitly to a header section since it's a repo change and not a package change